src/core/thread/mle.cpp

```diff
@@ -971,6 +990,30 @@ void Mle::SetMeshLocalPrefix(const MeshLocalPrefix &aMeshLocalPrefix)
 
 void Mle::ApplyMeshLocalPrefix(void)
 {
+    mLinkLocalAllThreadNodes.GetAddress().mFields.m8[3] = 64;
```

Comment:
nit: use [`Mle::MeshLocalPrefix::kLength`](https://github.com/openthread/openthread/blob/3535176535e5f027b8c8f65fb61fc272b9c4bfdb/src/core/thread/mle_types.hpp#L441)

Reply:
planned to submit a separate PR to apply `Mle::MeshLocalPrefix::kLength` and a common method for prefix-based multicast address.
Will append a patch to this PR.

Reply:
I'm fine to have a separate PR for this cleanup.

Reply:
ok. thanks.